src/redux/reducers/challenge.js

```diff
@@ -1,6 +1,10 @@
 import CNST from "../../constants";
 
 export default function reducer(state = {}, action) {
+  if (action.type === CNST.CHALLENGE.VOTE_CHALLENGE.SUCCESS) {
+    console.log(action);
+    console.log(state.data.challengeState);
+  }
```

Comment:
Maybe you`ve forgotten to remove console.log?

Reply:
fixed